CHANGE-NOTES.html

```diff
@@ -1,28 +1,20 @@
-<h3>v0.8.1</h3>
+<h3>v0.9.0</h3>
```

Comment:
@PawelLipski  anything else here?

Reply:
Nit: there are two sections now (unnamed and `Bug fixes/improvements`)... I'd split it to `Bug fixes` and `Improvements`

Reply:
Plus `Added support for IntelliJ 2021.1`.

Reply:
Not mentioning support for Android Studio yet coz I'm not sure if it's gonna work. I also haven't updated `It is compatible with all JetBrains products except Android Studio` in README.md for the same reason